index.js

```diff
@@ -37,14 +37,13 @@ const command = parsedArgs._[0];
 
 if (command === 'list-commands') {
   require('./commands/list-commands');
-  process.exit(0);
+} else {
+  exec(command)
+    .catch((err) => {
+      if (err && err.code === 'ENOENT') {
+        console.error(`Error: "${command}" is not an available ${cmd} command\n`);
+        showHelp();
+      }
+      process.exit(1);
```

Comment:
I just put this through to make it less likely in future: https://github.com/eslint/eslint/pull/7381